src/button/button.test.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env jest */
+const puppeteer = require('puppeteer')
+
+let browser
+let page
+let baseUrl = 'http://localhost:3000'
+
+beforeAll(async () => {
+  browser = await puppeteer.launch()
+  page = await browser.newPage()
+})
+
+describe('/components/button', () => {
+  describe('/components/button/link', () => {
+    it('renders a link button', async () => {
```

Comment:
Do we test for the anchor tag here, as well as in `button/template.test.js`, because here we're using Puppeteer with these tests? If we think we can't rely on `template.test.js`, should we also test for `href` inside this test as the next test will fail with a non helpful error if the `href` missing; this test would be the better place to catch that.

Reply:
I'm not sure this test is necessary at all since the next test would fail if a link did not render?

Reply:
removed